cpp/include/Ice/Proxy.h

```diff
@@ -319,12 +319,12 @@ namespace Ice
         /// @private
         void _iceI_ping(const std::shared_ptr<IceInternal::OutgoingAsyncT<void>>&, const Ice::Context&) const;
 
-        /// Gets the Slice interfaces supported by this object as a list of type IDs.
+        /// Gets the Slice interfaces supported by this object as a list of Slice type IDs.
```

Comment:
Again, we usually qualify `type IDs` with `Slice`. Just updated these 3 functions to follow that.